docs/advanced/mixins/mixins.md

```diff
@@ -34,17 +34,17 @@ Mixins will always be executed before the components.
 
 ```js
  <script>
-    import { exampleMixin } from '../mixins/exampleMixin.js'
+    import { exampleMixin } from '../mixins/exampleMixin.js';
     export default {
       mixins: [exampleMixin],
       created() {
         console.log('🤘🏼 from the component.')
       }
-    }
-    </script>
+    };
+</script>
 ```
 
-The output from that would we printing first the Mixin console and then the one from the component
+The output from that is we're showing first the Mixin console and then the one from the component
```

Comment:
Could you please describe this better,  **is _that_ we're showing**  like  similar to this:

_As a result we get:
1. The first mixing logs to the console
2. The component method logs to the console_

Reply:
fixed